src/Nethermind/Nethermind.Consensus/Processing/ProcessingStats.cs

```diff
@@ -200,8 +202,22 @@ public void UpdateStats(Block? block, IBlockTree blockTreeCtx, int recoveryQueue
                         > 75 => whiteText,
                         _ => ""
                     };
+
+                    var recoveryQueue = Metrics.RecoveryQueueSize;
+                    var processingQueue = Metrics.ProcessingQueueSize;
+
+                    string extraStats;
+                    if (recoveryQueue > 0 || processingQueue > 0)
+                    {
+                        extraStats = $"recover {recoveryQueue,5:N0} | process {processingQueue,5:N0}";
+                    }
+                    else
+                    {
+                        extraStats = $"exec code {resetColor} from cache {cachedContractsUsed,7:N0} |{resetColor} new {contractsAnalysed,6:N0}";
+                    }
```

Comment:
Hm... previously we created one string in `UpdateStats`, now we are creating 3.
I would in-line this into `_logger.Info()` call.

Reply:
Should we update stats outside the main thread?

Reply:
Moved to background thread. Doesn't use StringBuilder? Is 3 string per second isn't much in the scheme of things?

Reply:
Generating on background thread looks a good call

Reply:
But each string interpolation creates a StringBuilder?

Reply:
Post C#10 .NET6.0 is complicated https://devblogs.microsoft.com/dotnet/string-interpolation-in-c-10-and-net-6/
<img width="672" alt="image" src="https://github.com/NethermindEth/nethermind/assets/1142958/7e055315-3391-4d0d-a4e4-65c8f9c1e74f">

And it caches via ArrayPool withthe main allocations happening post Log.Info in NLogger

<img width="563" alt="image" src="https://github.com/NethermindEth/nethermind/assets/1142958/01216b37-1ba6-4e3b-b727-523017c95e87">